src/lib/slide-toggle/slide-toggle.ts

```diff
@@ -195,10 +206,13 @@ export class MatSlideToggle extends _MatSlideToggleMixinBase implements OnDestro
     // emit its event object to the component's `change` output.
     event.stopPropagation();
 
+    if (!this._dragging) {
+      this.toggleChange.emit();
+    }
     // Releasing the pointer over the `<label>` element while dragging triggers another
     // click event on the `<label>` element. This means that the checked state of the underlying
     // input changed unintentionally and needs to be changed back.
-    if (this._dragging) {
+    if (this._dragging || this.defaults.disableToggleValue) {
```

Comment:
Please update the comment to mention that we also revert the value back, if `disableToggleValue` is set to true.